HELPS/ClientApp/src/components/adminWorkshops/adminWorkshops.tsx

```diff
@@ -0,0 +1,104 @@
+import {
+    Collapse,
+    Container,
+    Navbar,
+    NavbarBrand,
+    NavbarToggler,
+    NavItem,
+    NavLink,
+    Form,
+    FormGroup,
+    Label,
+    Input,
+    Button
+} from 'reactstrap';
+import {AppState} from '../../types/store/StoreTypes';
+import {Dispatch} from 'redux';
+import {connect} from 'react-redux';
+import {Link} from 'react-router-dom';
+
+import * as React from 'react';
+import {Component} from 'react';
+import {
+    WorkshopEvent,
+    AdminWorkshopDispatchProps,
+    AdminWorkshopProps,
+    AdminWorkshopStateProps
+} from '../../types/components/AdminWorkshopsTypes';
+import {
+    addSkill
+} from '../../store/actions/AdminWorkshopActions';
+
+
+class AdminWorkshops extends Component{
+
+    componentDidMount(): void {
+        //this.props.retrieveWorkshops();
+    }
+
+    addSkillSet = (skill) => {
+       
+        adminWorkshops.dispatch({type: 'ADD_SKILL'
```

Comment:
Try not to dispatch directly from a component like this, move actions to their own file like you have with AdminWorkshopActions